upup/pkg/fi/cloudup/gcetasks/instancegroupmanager.go

```diff
@@ -63,6 +64,7 @@ func (e *InstanceGroupManager) Find(c *fi.CloudupContext) (*InstanceGroupManager
 	actual.Zone = fi.PtrTo(lastComponent(r.Zone))
 	actual.BaseInstanceName = &r.BaseInstanceName
 	actual.TargetSize = &r.TargetSize
+	actual.UpdatePolicy = &UpdatePolicy{MinimalAction: r.UpdatePolicy.MinimalAction, Type: r.UpdatePolicy.Type}
```

Comment:
can we make sure r.UpdatePolicy is not nil? otherwise we'll panic:

```go
if policy := r.UpdatePolicy; policy != nil {
	actual.UpdatePolicy = &UpdatePolicy{MinimalAction: policy.MinimalAction, Type: policy.Type}
}